yake/src/main.rs

```diff
@@ -0,0 +1,107 @@
+use std::path::PathBuf;
+
+use clap::{Parser, ValueEnum};
+use yake_rust::{Config, StopWords, Yake};
+
+#[derive(Clone, Copy, ValueEnum)]
+enum DedupFunction {
+    Level,
+    Jaro,
+    SeqM,
+}
```

Comment:
Could you tell, jaro is Jaro or Jaro-Winkler similarity? We are yet to port it.

Reply:
Removing in next commit.

Reply:
Wait wait, I can implement it, but I need more info haha 😄

Reply:
Will bring them back once they're implemented.